### PR TITLE
even more mertain icon changes

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
More icon tweaks
Small sawn martain fix

### CODE REVIEW

File: code/modules/projectiles/guns/projectile/automatic/solmarine.dm [hunk 1]
```diff
@@ -90,7 +90,7 @@
 
 	shotgun_haver = FALSE
 
-/obj/item/gun/projectile/automatic/omnirifle/omnicarbine/solmarine/shotgunless/sawn
+/obj/item/gun/projectile/automatic/omnirifle/omnicarbine/solmarine/shotgunless_sawn
```
Comment (nikothedude) [inline code/modules/projectiles/guns/projectile/automatic/solmarine.dm:93]: no reason to do this, a shotgunless solmarine rifle that isnt sawnoff might be made ion the future

Comment (Trilbyspaceclone) [reply code/modules/projectiles/guns/projectile/automatic/solmarine.dm:93]: Yes their is

Comment (nikothedude) [reply code/modules/projectiles/guns/projectile/automatic/solmarine.dm:93]: what reason then

Comment (Trilbyspaceclone) [reply code/modules/projectiles/guns/projectile/automatic/solmarine.dm:93]: Gun transmutations

**Pull request merged**